packages/react-ui-core/src/Photo/BackgroundPhoto.js

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(['BackgroundPhoto'], {
+  pure: true,
+})
+
+export default class BackgroundPhoto extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string,
+    fallbackUrl: PropTypes.string,
+    theme: PropTypes.object,
+    className: PropTypes.string,
+    alt: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  get fallback() {
+    const { fallbackUrl } = this.props
+    return fallbackUrl ? `url(${fallbackUrl})` : undefined
+  }
+
+  render() {
+    const { theme, className, url, alt, ...rest } = this.props
+    const image = [`url(${url})`].concat(this.fallback).filter(Boolean).join()
```

Comment:
This line took me a couple minutes to figure out what it was doing (because I'm a little slooow on the uptake).

Maybe split apart into something more readable?
```
joinUrlStyles(...urls) {
  return urls
    .filter(Boolean) // skip if undefined
    .map(url => `url(${url})`)
    .join() // defaults to comma-separated
```

Then `joinUrlStyles(url, this.fallback)`?

Reply:
I can do that if you'd like 😄

Reply:
More like

```
const joinUrlStyles = pipe(
  (...args) => ([...args]),
  filter(Boolean),
  map(url => `url(${url})`),
  join(','),
)
```

`joinUrlStyles(url, this.fallback)`